app/src/main/java/org/thoughtcrime/securesms/home/HomeActivity.kt

```diff
@@ -426,6 +428,24 @@ class HomeActivity : PassphraseRequiredActionBarActivity(),
             userDetailsBottomSheet.arguments = bundle
             userDetailsBottomSheet.show(supportFragmentManager, userDetailsBottomSheet.tag)
         }
+        bottomSheet.onCopyConversationId = onCopyConversationId@{
```

Comment:
it should probably be noted (or maybe removed) that we have the ability to copy the session ID in the `UserDetailsBottomSheet` itself as well (on long pressing the `publicKeyTextView`) but maybe we should get rid of that if this is a cross platform consistency thing. Or have both either way I don't mind just thought you should be aware in case you didn't know of it

Reply:
Yea I did see that but thought it would be odd to add 'Copy Group URL' and not have 'Copy Session Id' as well - it also comes off the back of a conversation about potentially overhauling the "user profile" (`UserDetailsBottomSheet`) page and content to standardise it across platforms so I wasn't sure what capabilities would be in there if/when it happened 🤷

Reply:
Yep makes sense, I don't think it hurts having it in there as well for now anyway. PR looks good anyway feel free to merge :+1: